chainer/functions/array/resize_images.py

```diff
@@ -169,11 +171,44 @@ def interpolate_grad_bilinear_gpu(gy, v, u, vw, uw, H, W):
     return gx.reshape((B, C, H, W))
 
 
+def compute_indices_and_weights(out_size, in_size, mode, align_corners, xp):
```

Comment:
Could align_corners and mode be unified in a single parameter?

mode = {'bilinear','bilinear_align','nearest'}

Reply:
I am ok with either implementation.
BTW, `align_corners` is quite common option in various DL frameworks. I thought it was easier for users if Chainer's implementation is consistent with others. Is that OK?

Reply:
Then lets keep that one!
Thanks for the clarification!